store/v2/options.go

```diff
@@ -27,6 +32,25 @@ type PruningOption struct {
 	Interval uint64
 }
 
```

Comment:
feels a bit odd to define arbitrary strings intended for CLI and config here but then use them in the server module

Reply:
yes, agreed, it should be in the server if it isn't used here.